bin/inference/pycbc_inference

```diff
@@ -208,6 +217,11 @@ with ctx:
     for key,val in static_args.iteritems():
         try:
             static_args[key] = float(val)
+            continue
+        except:
+            pass
+        try:
```

Comment:
Chris,

It can't get that far due to the continue. I'm not a fan of the nested
solution in case another parsing option is needed.

On Thu, Jul 28, 2016 at 7:54 PM, Christopher M. Biwer <
notifications@github.com> wrote:

> In bin/inference/pycbc_inference
> <https://github.com/ligo-cbc/pycbc/pull/994#discussion_r72669647>:
>
> > @@ -208,6 +217,11 @@ with ctx:
> >      for key,val in static_args.iteritems():
> >          try:
> >              static_args[key] = float(val)
> > +            continue
> > +        except:
> > +            pass
> > +        try:
>
> Maybe this should be nested or something. Its going to check if it can be
> typecast as a float and then a list of strings which isn't possible if the
> first is true.
>
> —
> You are receiving this because you were assigned.
> Reply to this email directly, view it on GitHub
> <https://github.com/ligo-cbc/pycbc/pull/994/files/faa8ef12bb117bd95e26a2c231acb248f0506cb2#r72669647>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/ACGrRiaJ32nIdxB2pY8v0d43PVuDvg2vks5qaOzUgaJpZM4JXaqA>
> .
>



-- 
Dr. Alexander Nitz
Max Planck Institute for Gravitational Physics (Albert Einstein Institute)
Callinstrasse 38
D-30167 Hannover,  Germany
Tel: +49 511 762-17097